tests/test_async_hundred_x_client.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+import respx
+from httpx import Response
+from decimal import Decimal
+from hundred_x.enums import OrderSide, OrderType, TimeInForce
+from hundred_x.async_client import AsyncHundredXClient
+from hundred_x.enums import Environment
+
+TEST_PRIVATE_KEY = "0x8f58e47491ac5fe6897216208fe1fed316d6ee89de6c901bfc521c2178ebe6dd"
+
+@pytest.fixture
+def client():
+    env = Environment.PROD
+    return AsyncHundredXClient(env, TEST_PRIVATE_KEY)
+
+BASE_URL = "https://api.100x.finance"
+
+@pytest.mark.asyncio
+@respx.mock
+async def test_get_symbol_info(client):
```

Comment:
id recommend inheiriting from the other tests cases, such that we know the async client implement all the methods of the sync client.

Could also use an abstract base class to enforce this.